src/workerd/api/node/buffer.c++

```diff
@@ -171,7 +101,7 @@ uint32_t writeInto(
       // Fall-through
     case Encoding::BASE64URL: {
       auto str = kj::str(string);
-      return base64_decode(
+      return nbytes::Base64Decode(
```

Comment:
potentially open for switching to simdutf?

Reply:
yup, I added a TODO for decoding, but handle them all in a follow up PR